content/definition.it.md

```diff
@@ -1,14 +1,44 @@
+
+* È distribuito sotto la la ["Licenza Ippocratica"](https://firstdonoharm.dev), o una licenza simile che proibisce la ridistribuzione del software in forma di lavoro modificata, derivata o raccolta dall'essere riconsegnata a qualsiasi individuo o organizzazione che facilita, incoraggia, manipola, costringe o costringe direttamente o indirettamente le persone a impegnarsi in comportamenti in conflitto con i diritti elencati nella [Dichiarazione universale dei diritti dell'uomo delle Nazioni Unite] (https://www.un.org/en/universal-declaration-human-rights/.
+
+* È distribuito sotto la [licenza "Anti 996"](https://996.icu/), o una licenza simile che proibisce la ridistribuzione del software in forma di lavoro modificata, derivata o raccolta a qualsiasi individuo o organizzazione che viola le [Convenzioni e / o raccomandazioni sugli standard internazionali del lavoro] (https://www.ilo.org/global/standards/introduction-to-international-labour-standards/conventions-and-recommendations/lang--en/index.htm) published by the International Labour Organization.
+
+* È distribuito sotto una [licenza "Atmosphere"](https://www.open-austin.org/atmosphere-license/), o una licenza simile che proibisce la ridistribuzione del software in lavori modificati, derivati ​​o raccolti forma per qualsiasi individuo o organizzazione che è materialmente e direttamente investito in operazioni o strutture che hanno un impatto negativo significativo sulla risposta climatica globale, come [documentato dal Segretariato dei cambiamenti climatici delle Nazioni Unite](https://unfccc.int/resource/climateaction2020/media/1308/unfccc_spm_2018.pdf).
+
+\* *Il termine "open source" in questo contesto si riferisce alla vasta esperienza di sviluppo di software pubblico e collaborativo in contributo ai beni comuni di software collettivi. Le licenze software ELOS possono essere o meno conformi ai [criteri convenzionali per le licenze software open source](https://opensource.org/osd-annotated).*
+
+I progetti Open Source con licenza etica sono incoraggiati a utilizzare il badge "Licensed Ethically" nella documentazione del loro progetto:
+
+![https://img.shields.io/badge/licensed-ethically-%234baaaa](https://img.shields.io/badge/licensed-ethically-%234baaaa) 
+
+[https://img.shields.io/badge/licensed-ethically-%234baaaa](https://img.shields.io/badge/licensed-ethically-%234baaaa)
+
+## Attribution and Contributions
+Mantenere questa definizione è il lavoro collaborativo del [Gruppo di lavoro sulla fonte etica](/apply). L'invio di Pull requests e la segnalazione di problemi nel nostro [repository](https://github.com/ethicalSource/ethicalsource.dev) sono incoraggiati. Un ringraziamento speciale è esteso a [Nate Berkopec](https://nateberkopec.com), [Will Barton](https://github.com/willbarton/), [Steve Phillips](https://tryingtobeawesome.com), e [Madalyn Parker](https://twitter.com/madalynrose).
```

Comment:
I would not translate the name of this group.

Reply:
In Italian I would structure the sentence as `Sono incoraggiati ...`